.rubocop.yml

```diff
@@ -31,6 +31,9 @@ Style/FrozenStringLiteralComment:
 RSpec/MultipleExpectations:
   Enabled: false
 
+RSpec/DescribeClass:
```

Comment:
Allows us to name integration specs by the behavior they test rather than the class being used to test said behavior